pkg/prometheus/statefulset.go

```diff
@@ -34,8 +34,7 @@ import (
 
 const (
 	governingServiceName            = "prometheus-operated"
-	DefaultPrometheusVersion        = "v2.7.1"
-	DefaultThanosVersion            = "v0.8.1"
+	DefaultThanosVersion            = "v0.10.0"
```

Comment:
This should be v0.10.1 since there is a bug in thanos sidecar v0.10.0

Reply:
Thanks, I'll update this.